specification/arrays.md

```diff
@@ -0,0 +1,137 @@
+An array is a numbered, fixed-size sequence of elements of a single [type](link to type spec), called the element type. The element type of an array can be any Rust [type](link to type spec). 
```

Comment:
Does the size of the type of an array need to be known at compile time? I am not sure that this is even applicable, but I thought that I would note it.

Reply:
cf https://spec.ferrocene.dev/types-and-traits.html#syntax_arraytypespecification

```
The [element type](https://spec.ferrocene.dev/glossary.html#term_element_type) shall be a [fixed sized type](https://spec.ferrocene.dev/glossary.html#term_fixed_sized_type).
```

where it says that the *element type* needs to have a fixed size.